deploy/start.go

```diff
@@ -198,9 +194,33 @@ func RunTests(name string, ts []T) {
 			continue
 		}
 
-		rs[i] = RunStatsAvg(runs)
+		s, err := AverageStats(runs...)
+		if err != nil {
+			dbg.Fatal("Could not average stats for test ", i)
+		}
+		rs[i] = s
+		rs[i].WriteTo(f)
+		// Write the header if you still havent done it
+		if !headerWritten {
```

Comment:
you saw that I did some work on my ‘restructure_main’-branch - I think it would be good to go on from there.

> On 30 Sep 2015, at 16:06, nikkolasg <notifications@github.com> wrote:
> 
> In deploy/start.go <https://github.com/DeDiS/cothority/pull/7#discussion_r40797007>:
> 
> > @@ -198,9 +194,33 @@ func RunTests(name string, ts []T) {
> >  			continue
> >  		}
> >  
> > -		rs[i] = RunStatsAvg(runs)
> > +		s, err := AverageStats(runs...)
> > +		if err != nil {
> > +			dbg.Fatal("Could not average stats for test ", i)
> > +		}
> > +		rs[i] = s
> > +		rs[i].WriteTo(f)
> > +		// Write the header if you still havent done it
> > +		if !headerWritten {
> Yes. I can take the first test in the array of tests, then get a stat to write the header.
> We'll need to refactor these "tests arrays" anyway later as they horribly lack of usability and checkability features (for example, check that all tests are using the same "app")
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/DeDiS/cothority/pull/7/files#r40797007>.
>